fc/mef.py

```diff
@@ -539,54 +261,67 @@ def sc_fun(p,x):
         return np.exp(p[0] * x + p[1])
     
     # Fit parameters
-    err_par = lambda p: err_fun(p, peaks_ch, peaks_mef)
+    err_par = lambda p: err_fun(p, fl_channel, fl_mef)
     res = minimize(err_par, params, options = {'gtol': 1e-6})
 
     # Separate parameters
-    sc_params = res.x
+    beads_params = res.x
 
     # Beads model function
-    sc_beads = lambda x: fit_fun(sc_params, x)
+    beads_model = lambda x: fit_fun(beads_params, x)
 
     # Standard curve function
-    sc = lambda x: sc_fun(sc_params, x)
+    std_crv = lambda x: sc_fun(beads_params, x)
     
-    return (sc, sc_beads, sc_params)
-
-def plot_standard_curve(peaks_ch, 
-                        peaks_mef,
-                        sc_beads,
-                        sc_abs,
-                        xlim = (0., 1023.),
-                        ylim = (1, 1e8)):
+    return (std_crv, beads_model, beads_params)
+
+def plot_standard_curve(fl_channel,
+                        fl_mef,
+                        beads_model,
+                        std_crv,
+                        xlim=(0.,1023.),
+                        ylim=(1.,1e8)):
```

Comment:
Actually, `xlim` is explicitly set when the function is called from `get_transform_fxn`. The default value could be eliminated and there would be no difference. Letting matplotlib figure out the appropriate default xlim is a good idea, though.

For `ylim`, I looked at our typical MEF values and saw that they go from ~1e2 to ~1e6. Adding two decades on each side results in the values that I used. If you let matplotlib figure out the limits, you would end up with different limits when you change the gain, which I don't like very much.